### PR TITLE
New package: Sentry.sentry-cli version 2.18.0

### PR DESCRIPTION
**Pull request by superusercode**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107669)

### CODE REVIEW

File: manifests/s/Sentry/sentry-cli/2.18.0/Sentry.sentry-cli.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Sentry.sentry-cli
+PackageVersion: 2.18.0
+InstallerType: portable
+ReleaseDate: 2023-05-19
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/getsentry/sentry-cli/releases/download/2.18.0/sentry-cli-Windows-x86_64.exe
+  InstallerSha256: 23EBC233A2E085249B0BFEFF2B11617E6212D81525363978FBFA5679687881A2
```
Comment (Trenly) [inline manifests/s/Sentry/sentry-cli/2.18.0/Sentry.sentry-cli.installer.yaml:11]:
```suggestion
  InstallerSha256: 23EBC233A2E085249B0BFEFF2B11617E6212D81525363978FBFA5679687881A2
  Commands: 
  - "sentry"
```

**Review by Trenly — COMMENTED** [review]
--

**Review by Trenly — COMMENTED** [review]
This package has a runtime dependency
image

**Comment by Trenly** [comment]
> @Trenly is `PortableCommandAlias` usable with standalone exes?

No, but `Commands` does the same thing. For standalone exes, the commands list is validated to be at most 1 command, and that command is used as the alias

**Review by stephengillie — APPROVED** [review]

**Pull request merged**